client/lib/performance-tracking/api.js

```diff
@@ -0,0 +1,43 @@
+const getLux = () => {
+	const lux = window.LUX || {};
+	// Reassign it so changes in LUX are picked when Lux starts
+	window.LUX = lux;
+
+	// Don't send the first navigation automatically, we'll do it.
+	lux.auto = false;
+	lux.debug = true;
+	return lux;
+};
+
+const addMetadata = ( metadata ) => {
+	const lux = getLux();
+
+	Object.entries( metadata ).forEach( ( [ key, value ] ) => {
```

Comment:
Are there any safety guards we need to put against this, like making sure to blacklist certain types of information (payment info, PPI in general?)

Reply:
How can we detect that kind of information automatically?

I can add a comment asking the dev to ensure there is no private information.